src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfacePrompt.cs

```diff
@@ -306,7 +306,8 @@ public override
                         null,   // caption already written
                         null,   // message already written
                         null,
-                        string.Empty);
+                        string.Empty,
+                        confirmPassword: false);
```

Comment:
Please do the same for all arguments for consistency.

Reply:
Can do, but those were already there before I made any changes. Do you want me to include those changes here as well?

Reply:
Yes, I see. My request is to get consistency. This makes a code more readable.

Reply:
So what do we want to do with this? Change it back to just `false` for consistency or leave it at `confirmPassword: false`?

Reply:
Please add names for all, not only you changed, constants.